CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+x.x.x Release notes (yyyy-MM-dd)
+=============================================================
+### Enhancements
+* None.
+
+### Fixed
+* Rare crash when a schema was updated ([realm/realm-cocoa#6680](https://github.com/realm/realm-cocoa/issues/6680))
```

Comment:
Normally we should aim to describe how the user can see observe the crash so it's easy for them to see if this fixes something they have seen. E.g. here I would just add "Rare crash (in realm::Table::migrate_links)". But I assume we haven't had any JS users experience this?